src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+interface ITextFieldProps {
+  customClassNames?: string;
+  value?: string;
+  label?: string;
+  type: string;
+  name?: string;
+  placeholder?: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+function TextField(props: ITextFieldProps) {
+  return (
+    <label>
+      {props.label}
+      <input
+        className={`${props.customClassNames}`}
```

Comment:
Since `customClassNames` could possibly be undefined or null, we could use a ternary here to replace any undefined / null with an empty string.
```suggestion
        className={`${props.customClassNames ? props.customClassNames : "" }`}
```